src/main/java/io/stargate/sgv2/jsonapi/exception/mappers/ThrowableToErrorMapper.java

```diff
@@ -219,7 +223,7 @@ private static CommandResult.Error handleJsonProcessingException(
           .toApiException(
               "\"%s\" not one of known fields (%s) at '%s'",
               upe.getPropertyName(), knownDesc, upe.getPathReference())
-          .getCommandResultError(Response.Status.OK);
+          .getCommandResultError();
```

Comment:
so this defaults to 200?

Reply:
Yes; it uses `Response.Status` from `JsonApiException`, which in turn defaults to `Response.Status.OK`.